source/docs/software/dashboards/dashboard-intro.rst

```diff
@@ -0,0 +1,42 @@
+Choosing a Dashboard
+====================
+
+A dashboard is a program used to retrieve and display information about the operation of your robot.  There are two main types of dashboards that teams may need: driver and programmer dashboards.  Some dashboards will try to accommodate both purposes.
+
+Driver Dashboard
+----------------
+
+During competition the drive team will use this dashboard to get information from the robot.  It should focus on conveying key information instantly.  This is often best accomplished by using large, colorful, and easy to understand visual elements.
+
+Most teams will use this dashboard to select their autonomous routine.  It is often helpful to hide this choice after the selection is made as that information is not needed during the match.
```

Comment:
The part about hiding the autonomous selection seems out of place. I'm also not sure how to do it except in Shuffleboard by selecting another tab. Recommend removing from here, and possibly adding to individual dashboard articles with more details if applicable.